docs/docfx.json

```diff
@@ -51,22 +51,15 @@
     },
     "fileMetadata": {
       "ms.prod": {
-        "concepts/*.md": "product-specific",
-        "design/*.md": "product-specific",
         "develop/*.md": "product-specific",
-        "excel/*.md": "excel",
         "overview/*.md": "product-specific",
-        "publish/*.md": "product-specific",
-        "reference/*.md": "product-specific",
-        "reference/manifest/*.md": "product-specific",
         "resources/*.md": "product-specific",
         "testing/*.md": "product-specific"
```

Comment:
Let's change these "product-specific" tags to "excel". We don't have application-agnostic pages like in Office Add-ins.